src/qiskit_sphinx_theme/assets/styles/_api.scss

```diff
@@ -76,7 +94,7 @@ dl[class]:not(.option-list):not(.field-list):not(.footnote):not(.glossary):not(.
 // section headers like "Attributes" and "Methods". We disagree with the design in
 // https://github.com/pradyunsg/furo/discussions/505. Instead, use the rules from `p.rubric`.
 dl.py dd p.rubric {
-  font-size: 1.125em;
+  font-size: 125%;  // We make this bigger than p.rubric to match the bigger font sizes for the page.
```

Comment:
I have a feeling that using percentage for font size is not best practice and it's better to use rem, but not confident. I think it depends if we want the size to be relative to the parent element or relative to the root element

Reply:
I went with percentage because I realized it's what Furo itself is using. So be consistent.